### PR TITLE
pillow 3.1.0

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by bfontaine** [comment]
Audit is failing and that’s expected; but the failing test is not:
```
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20160111-17569-85lgs.sb /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Library/Homebrew -- /usr/local/Library/Homebrew/test.rb /usr/local/Library/Taps/homebrew/homebrew-python/pillow.rb --verbose
==> python -m nose Tests/test_*
..................SSSSSSS.............SSSSSSSSSS............SS.............................................SSSSSSSSSSSSSSSSSSS.....................................S...........................................................................................................S....................................................................................................................................S.....................................S...................................................SS...........SSSSSS............................S.E..SS.......
======================================================================
ERROR: Test_scipy_resize.test_imresize4
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/tmp/pillow20160111-17570-148rkq7/Tests/test_scipy.py", line 40, in test_imresize4
    im4 = misc.imresize(im, 200, mode='F')  # percentage
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/scipy/misc/pilutil.py", line 420, in imresize
    imnew = im.resize(size, resample=func[interp])
  File "/usr/local/lib/python2.7/site-packages/PIL/Image.py", line 1538, in resize
    size=tuple(size)
TypeError: 'float' object is not iterable
```

File: pillow.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class Pillow < Formula
+  desc "Friendly fork of the Python Imaging Library"
   homepage "https://github.com/python-imaging/Pillow"
```
Comment (dunn) [inline pillow.rb]: Redirects to https://github.com/python-pillow/Pillow FYI

Comment (bfontaine) [inline pillow.rb]: Thanks!

**Comment by tdsmith** [comment]
I think the root cause here is that the system scipy is too old? We could depend on our scipy, I guess, or we could test without scipy, since those features are optional.

**Pull request closed without merging**